styles/index.css

```diff
@@ -93,48 +87,51 @@ button:hover {
   transition: 0.2s;
 }
 
-.spaced-items {
-  display: flex;
-  flex-direction: row;
-  align-content: center;
-  justify-content: space-around;
-  padding-top: 2%;
-  padding-bottom: 2%;
+/* ........ header css done........... */
+
+.info {
+  margin: 10px;
 }
 
-.centered-items {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  padding-top: 2%;
-  padding-bottom: 2%;
+.info h2 {
+  text-align: center;
+  padding: 20px;
+  font-size: 40px;
 }
 
-.top-divider {
-  border-style: solid none none none;
+.info p {
+  justify-content: center;
+  text-align: center;
+  font-size: 1.2em;
+  line-height: 1.5em;
+  margin: 20px;
 }
 
-.footer {
-  font-size: 25px;
-  text-transform: uppercase;
-  margin-top: 50px;
-  color: #f0fff0;
+.row-images {
+  display: flex;
+  justify-content: space-around;
 }
 
 .row-images img {
-  display: inline-flex;
-  padding-left: 40px;
-  padding-top: 50px;
   width: 23%;
-  height: 20%;
 }
 
-.main-article {
-  font-family: "Quicksand", sans-serif;
-  font-size: 27px;
+.donation {
+  margin-top: 30px;
+}
+
+footer {
```

Comment:
you can give text color white for the footer as background color for footer is black